### ISSUE TITLE
Train loss does not converge

### ISSUE THREAD

**Issue by CSJasper** [xljh0520/JOTR#8]
I would like to express my appreciation for your work. But the train loss does not converge using the code uploaded. (After training MPJPE is about 550). Could you check if there is any problem with the code.

**Comment by 124518** [comment]
Hello, have you solved this problem?

**Comment by xljh0520** [comment]
Sorry for the late reply. Could you kindly let me know how many GPUs you are using for training and what the total batch size is?

**Comment by CSJasper** [comment]
Hi, thanks for reply. 

I train this model using 2 RTX 3090 gpus. And use exactly same yaml file in 'assets/yaml’. The losses in the experient ‘main_train’ in the train.sh script do not converge when I ran that code.

Best, Chan Hee Kang.

**Comment by xljh0520** [comment]
Thank you for your interest in our work. Based on your description, it seems that the issue may be due to a mismatch between the batch size and the number of GPUs. It appears that your total batch size is 2 * 32. In my experiments, I consistently maintained a total batch size of 256 in the main training part. I was able to reproduce the results using 8 V100 GPUs (8 * 32) and 4 A100 GPUs (4 * 64). If you have fewer GPUs available, you can try techniques such as gradient accumulation or reducing the learning rate to simulate a larger batch size.
